### ISSUE TITLE
Screenshot light version

### ISSUE THREAD

**Issue by edouard-lopez** [rafaelrinaldi/pure#113]
Following merge of #103 the screenshot has been updated to reflect colors. However, it is a dark version:

![screenshot](https://user-images.githubusercontent.com/1212392/51090701-3b521a80-1780-11e9-8be0-09ab94886e7a.png)

![screenshot](https://user-images.githubusercontent.com/1212392/51090703-4147fb80-1780-11e9-855a-28dbd457bf0b.png)

We need an updated light version.

@boyeborg Could you provide one?

**Comment by rafaelrinaldi** [comment]
I think we can just have the dark one. It looks great already! No need for the light version.

**Comment by edouard-lopez** [comment]
It's good to know when a theme/prompt works with the colour scheme you prefer.

I work on the light scheme, but with `tmux` my colours seem to be a bit different, also I don't remember the site used to create the image :/

**Comment by edouard-lopez** [comment]
Can't get a result similar to what @boyeborg generated. What language did you use? 

I use this content:

```fish

~
❯ sleep 15s; false

~ 15s
❯ cd pure

/data/projects/pure master
❯ git reset HEAD~1 -q

/data/projects/pure master* ⇣
❯ _
```

**Comment by rafaelrinaldi** [comment]
Closed by #132
